src/extensions/mockComponentPartial.ts

```diff
@@ -0,0 +1,32 @@
+import {
+    BrsType,
+    ValueKind,
+    Callable,
+    StdlibArgument,
+    BrsInvalid,
+    RoAssociativeArray,
+} from "../brsTypes";
+import { Interpreter } from "../interpreter";
+
+export const mockComponentPartial = new Callable("mockComponentPartial", {
```

Comment:
I think it might make more sense to call this `mockFunctions` to align with the existing `_brs_.mockFunction`